.github/styles/config/vocabularies/Polkadot/accept.txt

```diff
@@ -41,11 +54,14 @@ Rococo
 [Rr]untimes
 Shiden
 Smoldot
+Snowbridge
+Snowfork
 Solochain
 Subscan
 [Ss]ubxt
 [Ss]udo
 TxWrapper
+Wasm
+WebAssembly
```

Comment:
We should stick to just one term, not all of them, like WASM, Wasm, and WebAssembly. I would suggest choosing one and keeping it consistent throughout the docs

Reply:
The Vale updates I push this evening will include "Use (blank) instead of (blank)" for wasm, WASM, and WebAssembly to all suggest Wasm as the correct usage.